blogs/2020/07/27/containers-edu.md

```diff
@@ -186,12 +186,10 @@ Now that you have a dev container, you can upload it to a repository so that you
 
 Make sure you have [Git installed](https://git-scm.com/downloads). You can [initialize a new Git repository](https://code.visualstudio.com/docs/editor/versioncontrol#_initialize-a-repository) from your container project in VS Code. Alternatively, you can use `git init` from your project's directory in the command line.
 
-Following the guide [here](https://kbroman.org/github_tutorial/pages/init.html), the next step is to connect your local repository to a [new repo](https://github.com/new) from the GitHub desktop client:
+Following [this guide](https://kbroman.org/github_tutorial/pages/init.html), the next step is to connect your local repository to a [new repo](https://github.com/new) from the GitHub desktop client:
```

Comment:
You could even go further and say:
"Following this [Start a new git repository guide]()..."
That way someone using a screen reader knows exactly what to expect if they follow the link.

Reply:
That's even better! Updated.